src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparatorNonPackageInfoWithJavadocBeforePackage.java

```diff
@@ -12,7 +12,7 @@
 
 /** Comment1
  Comment2 */
-package com.puppycrawl.tools.checkstyle.checks.whitespace.emptylineseparator; // violation
+package com.puppycrawl.tools.checkstyle.checks.whitespace.emptylineseparator; // violation ''package' should be separated from previous line.'
```

Comment:
Please put violation comment on line bellow.
`// violation above ...`
Do same for all others

Reply:
@romani I have added violation messages above in some of the files because adding them below results in creating more violations.

Reply:
Adding violation messages on different lines instead of the violated line is causing some errors. I am trying to resolve them, but without success so far.

Reply:
the expected line that was shown in the errors are diff that what we have in EmptyLineSeparatorChecktest.java. 
Same for the CLI, the violation lines are diff than the expected lines.

Reply:
>Same for the CLI, the violation lines are diff than the expected lines.

Please always share CLI output if in doubt, same pattern as check documentation update, bug report template, etc. Show config, compilation of input file, and Checkstyle execution.

Reply:
Above translates to “one line above”, search for other examples in the project of “x lines above” syntax.

Reply:
@nrmancuso Even though I have updated the EmptyLineSeparatorCheckTest.java file, I am still getting some errors. The violation line in the CLI output is different from the expected violation line shown in the error message.

Reply:
This check do care about anything above and below target line.

Please restore to state as it was before my request. It is exceptional Check, it ok to have long lines

Reply:
A comment like:
`// violation above ''package' should be separated from previous line.'`

tells parser to expect violation on line directly above. BDD comment should be:

`// violation 2 lines above ''package' should be separated from previous line.'

Reply:
@nrmancuso  , I understand your point. As @romani  mentioned, this check is concerned with whether the target line is above or below. That's why I initially didn't put the violation message on a separate line, as it would lead to more violations.